### ISSUE TITLE
Update to reference latest version of Newtonsoft.Json library

### ISSUE THREAD

**Issue by nick-goloborodko** [Windows-XAML/Template10#1737]
I'm stuck in a bit of DLL-hell because of Template10 still uses older version of Newtonsoft.Json library (v11.0.2 i think?) and a few of my other reference libraries require v12+. Any chance of updating this reference?

**Comment by dmitrymal** [comment]
We had to downgrade the Newtonsoft version in all our dependencies to 10.0.3
```
    <PackageReference Include="Newtonsoft.Json">
      <Version>10.0.3</Version>
    </PackageReference>
```
:(

**Comment by darenm** [comment]
Of course, the source code for T10 is available, so you can download that and upgrade it to whatever version of Newtonsoft you wish to use.
